app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_cp, read_from, write_to = command.split()
```

Comment:
What if command split returns more then 3 or less then 3 elements?
Also source_file / destination file would be more appropriate names.